tests/Feature/Http/Controllers/TaskControllerTest.php

```diff
@@ -49,16 +52,34 @@ public function test_create_is_successful(): void
         $response = $this->json('POST', self::URI, [
             'name' => 'test task',
             'description' => 'this is a test task',
-            'status' => 'on progress'
+            'status' => 'on progress',
+            'comment' => [
+                'title' => 'comment title',
+                'body' => 'comment body'
+            ],
         ]);
+
         $expected = [
             'name' => 'test task',
             'description' => 'this is a test task',
-            'status' => 'on progress'
+            'status' => 'on progress',
+            'comments' =>[ [
+                'title' => 'comment title',
+                'body' => 'comment body'
+            ]],
         ];
+
         $response->assertStatus(201)
-             ->assertJsonFragment($expected);
-        $this->assertDatabaseHas('tasks',$expected);
+            ->assertJsonFragment($expected);
+        $this->assertDatabaseHas('tasks', [
+                    'name' => $expected['name'],
+                    'description' => $expected['description'],
+                    'status' => $expected['status']]
+                );
+        $this->assertDatabaseHas('comments', [
+            'title' => $expected['title'],
+            'body' => $expected['body']
```

Comment:
Whether expected array has `title` and `body` key? Write down what is expecting here..
```
'title' => 'comment title',
'body' => 'comment body',
```